File: app/src/main/java/com/example/lucas/haushaltsmanager/Entities/Currency.java
package com.example.lucas.haushaltsmanager.Entities;

import androidx.annotation.NonNull;

// TODO: Kann ich die Currency durch die Java.util implementierung der Currency austauschen?
public class Currency {
    @NonNull
    public String getName() {
        return "Euro";
    }

    @NonNull
    public String getShortName() {
        return "EUR";
    }

    @NonNull
    public String getSymbol() {
        return "€";
    }
}

File: app/src/main/java/com/example/lucas/haushaltsmanager/entities/Currency.kt
package com.example.lucas.haushaltsmanager.entities

import com.example.lucas.haushaltsmanager.App.app

/**
 * TODO: Remove this class if no errors are reported regarding the Java.Currency implementation
 */
class Currency {
    fun getName(): String {
        return app.getDefaultCurrency().displayName
    }

    fun getShortName(): String {
        return app.getDefaultCurrency().currencyCode
    }

    fun getSymbol(): String {
        return app.getDefaultCurrency().symbol
    }
}